bin/run-test

```diff
@@ -1,25 +0,0 @@
-#!/bin/bash
```

Comment:
This is a good reminder that I never quite aligned the CI/CD for this repo completely with [the overall guidance for a Harmony repository](https://wiki.earthdata.nasa.gov/display/HARMONY/Harmony+Services+General+Repository+Structure). I was a bit worried that removing this would break the CI/CD, [but looks like the script being used is bin/test](https://github.com/nasa/harmony-netcdf-to-zarr/blob/main/.github/workflows/run_tests.yml#L38).

I think that means this change makes sense, but we (I) probably need to come in here at some point and make sure all the scripts have names that are consistent with our guidance. It _seems_ like it's primarily just a minor renaming thing to get there, and out of scope for this PR, for sure.